src/Utils.cpp

```diff
@@ -141,7 +141,7 @@ utils::replaceEmoji(const QString &body)
         if (!insideTag && utils::codepointIsEmoji(code)) {
             if (!insideFontBlock) {
                 fmtBody += QStringLiteral("<font face=\"") % UserSettings::instance()->emojiFont() %
-                           QStringLiteral("\">");
+                           QStringLiteral("\" size=\"4\">");
```

Comment:
This should really be dependent on the large emoji setting instead of forced on everyone.

Reply:
Hm, that is a good observation, but I think there are at least 2 reasons that that should not be dependent:

1. The emoji by default are nearly undecipherable in many cases where they are not upscaled (although ymmv if you have increased your overall font size).
2. This would require us to reprocess all displayed messages every time the large emoji setting was changed, which probably would be overkill.

Reply:
Although if you think there is a good way to quickly reprocess fonts or if it is fine to not resize the emoji until a restart has happened, then I have a patch that I can add to this PR for that.

Reply:
Well, I don't want larger emoji than my text size. It just looks wrong to me. Imo just reusing the setting for the large emoji makes the most sense then, to not add too many toggles. I don't really care, if that requires a restart (or switching rooms) :)

Reply:
Fair enough then. I have toyed with the idea of having an indicator for settings that require a restart, so I could maybe add that sometime to make this obvious.